### ISSUE TITLE
should eval_step return targets?

### ISSUE THREAD

**Issue by shahules786** [flowerpot-ai/stabilizer#19]
In https://github.com/flowerpot-ai/stabilizer/blob/master/stabilizer/trainer.py#L17 function, it is return targets as one of the outputs, is it necessary because the function receives the same as one of its inputs. @VigneshBaskar

**Comment by VigneshBaskar** [comment]
@shahules786 Yes it doesn't make sense to return targets. I will remove it and create a PR
